### ISSUE TITLE
Gear icon obscured by mac buttons

### ISSUE THREAD

**Issue by chenglou** [rsms/fb-mac-messenger#309]
<img width="326" alt="screenshot 2016-07-18 21 06 25" src="https://cloud.githubusercontent.com/assets/1909539/16938330/810f7f44-4d2b-11e6-85b8-2ad9b18f755e.png">

Btw, I'm back on working on messenger.com. Unfortunately we can't always sync our changes with yours, but I'll try to provide help on these layout issues whenever I can. Just ping me (no promise though!)

**Comment by jake32321** [comment]
Yeah I just ran into the same issue.  Is there any progress being made on this?
